playground/nextjs-app-router/components/demo/Swap.tsx

```diff
@@ -45,9 +45,9 @@ function SwapComponent() {
   };
 
   const usdcToken: Token = {
-    name: 'USDC',
+    name: 'USD',
     address: '0x833589fcd6edb6e08f4c7c32d4f71b54bda02913',
-    symbol: 'USDC',
+    symbol: 'USD',
     decimals: 6,
     image:
```

Comment:
Should revert this or else our Swap component won't show USDC as needed:

![Uploading CleanShot 2024-12-18 at 12.57.24@2x.png…]()

Reply:
Been testing it and forgot to revert. Thank you for catching!